raidpackage.py

```diff
@@ -137,19 +117,31 @@ async def process_user_quantity_input(client, usr, item, qtyMax, usr_id):
     botMsg = await usr.send(embed=qtyEmbed)
 
     def checkMsg(msg):
-        try:
+        if msg.content != "":
             print(msg.content)
             qty = int(msg.content)
             return True
-        except Exception:
-            raise Exception("mismatch type in ProcessUserQtyInput")
-
+                    
     try:
         msg = await client.wait_for("message", timeout=20.0, check=checkMsg)
     except asyncio.TimeoutError:
         await botMsg.delete()
         msg = await client.wait_for("message", timeout=20.0, check=checkMsg)
         await cancel_order(msg, usr_id, client)
+    
+    # non-numeric string
+    except TypeError:
+        error_msg = await usr.send(f"You must enter a number - please unclick and reclick the {item} emoji")
+        _error_messages.append(error_msg)
+    except ValueError:
+        await botMsg.delete()
+        error_msg = await usr.send(f"You must enter a number - please unclick and reclick the {item} emoji")
+        _error_messages.append(error_msg)
     else:
         await botMsg.delete()
+        
+        if len(_error_messages) > 0:
+            for error_msg in _error_messages:
+                await error_msg.delete()
+                _error_messages.remove(error_msg)
         return int(msg.content)
```

Comment:
Here you are not checking if `int(msg.content) <= maxQty`

Reply:
Fixed in commit 1f34b26c96d2f7f135f263d8981e735e55197e61